hostplumber/controllers/hostconfig_controller.go

```diff
@@ -132,7 +224,8 @@ func applySriovConfig(sriovConfigList []plumberv1.SriovConfig) error {
 				sriovutils.EnableDriverForVfs(pfName, "ixgbevf")
 			}
 
-			if sriovConfig.MTU != nil && *sriovConfig.MTU > 0 {
+			if sriovConfig.MTU != nil && *sriovConfig.MTU >= 576 {
+				log.Info("sriovConfig MTU", "MTU", *sriovConfig.MTU)
```

Comment:
Seems like a left-over debug statement. If not, it should probably be made into a bit more readable log message ;)